src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -266,6 +266,11 @@ def load_arguments(self, _):
                    help='Provide a string value of a Storage Account in the provided Resource Group. Or Resource ID of a Storage Account in a different Resource Group')
         c.argument('consumption_plan_location', options_list=['--consumption-plan-location', '-c'],
                    help="Geographic location where Function App will be hosted. Use 'functionapp list-consumption-locations' to view available locations.")
+        c.argument('is_linux', action='store_true', required=False, help='host webapp on Linux worker')
+        c.argument('zip_url', help='Url string to zip location The content from this location is deployed to the function app')
```

Comment:
The wording is slightly misleading. The content are not deployed to the app. The app will download and run from that zip every time it starts up. To me it feels that the former suggests that I can delete that zip after deployment is done, which is not true

Reply:
i see.  Does this need to be a Storage SAS URL? if so how about
SAS URL for the blob location of the code. Should we also add link to some documentation on Run -as -zip?

Reply:
@panchagnula is that the part for deploy from Zip? or Run from Zip (I thought we are not going to do this  until after build)

Reply:
This is run from zip, that is what  functions is using here. There nothing to do here for CLI the url is a  SAS URL, that is set as a app setting.